docs/src/model_creation/conservation_laws.md

```diff
@@ -39,7 +39,7 @@ Using the `unknowns` function we can confirm that the ODE only has a single unkn
 ```@example conservation_laws
 unknowns(osys)
 ```
-Next, using `parameters` we note that an additional parameter, `Γ[1]` has been added to the system:
```

Comment:
Due to MTK updates, the conservaton law parameters are not displayed as `Γ[1:1]` anymore, but only `Γ`.
(I prefered the old way and have raised an issue about it)